app/javascript/components/MainApp.js

```diff
@@ -49,7 +50,7 @@ class MainApp extends React.Component {
       </Nav>
       <Switch> 
         <Route path="/" exact component={Home} /> 
-        <Route path="/profile" exact component={Profile} /> 
+        // <Route path="/profile" exact render={( ...props) => <Profile edit_user_route={edit_user_route}/> } /> 
```

Comment:
Is this intended to be commented out. If so use JSX comment syntax: {/* <your code here> */}